superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FiltersConfigModal.tsx

```diff
@@ -208,7 +230,11 @@ export function FiltersConfigModal({
     if (values) {
       createHandleSave(
         filterConfigMap,
-        filterIds,
+        filterIds
+          .slice()
+          .sort(
+            (a, b) => orderedFilters.indexOf(a) - orderedFilters.indexOf(b),
+          ),
```

Comment:
This code is duplicated in 3 places in this file. Could you extract it to a function?